clang/test/dpct/cmake_migation/case_011/input.cmake

```diff
@@ -0,0 +1,25 @@
+add_library(culib)
+add_executable(cuexe)
+TARGET_COMPILE_FEATURES(culib PUBLIC c_std_11)
+target_compile_features(culib PRIVATE cxx_std_11)
+target_compile_features(culib 
+  PUBLIC 
+  cuda_std_11)
+
+TARGET_COMPILE_FEATURES(culib PUBLIC c_std_17)
+target_compile_features(culib PRIVATE cxx_std_17)
+target_compile_features(culib 
+  PUBLIC 
+  cuda_std_17)
+
+TARGET_COMPILE_FEATURES(cuexe PUBLIC c_std_20)
+target_compile_features(cuexe PRIVATE cxx_std_20)
+target_compile_features(cuexe 
+  PUBLIC 
+  cuda_std_20)
+
+TARGET_COMPILE_FEATURES(cuexe PUBLIC c_std_23)
+target_compile_features(cuexe PRIVATE cxx_std_23)
+target_compile_features(cuexe 
+  PUBLIC 
+  cuda_std_23)
```

Comment:
For this part, you can commented as a TODO here, as it should be untouched, just like https://github.com/oneapi-src/SYCLomatic/blob/8eafeeafbdc6a04021db4acb32351fc6904b8ed2/clang/test/dpct/cmake_migation/case_021/input.cmake#L7C3-L7C4

Reply:
Added comments as suggested